lib/Elevate/Blockers/Repositories.pm

```diff
@@ -10,7 +20,21 @@ use parent qw{Elevate::Blockers::Base};
 
 use Log::Log4perl qw(:easy);
 
+# still used by disable_known_yum_repositories function
 use constant DISABLE_MYSQL_YUM_REPOS => qw{
```

Comment:
I'm confused why this list exists. Didn't we move to being repo name based not file based?